parts/windows/windowsazurecnifunc.ps1

```diff
@@ -68,11 +68,21 @@ Set-AzureCNIConfig
         $KubeServiceCIDR,
         [Parameter(Mandatory=$true)][string]
         $VNetCIDR,
-        [Parameter(Mandatory=$true)][string]
-        $TargetEnvironment
+        [Parameter(Mandatory=$true)][bool]

```

Comment:
Missing 
```
        $IsAzureStack,
        [Parameter(Mandatory=$true)][bool]
```

Reply:
We don't support AzureStack, removed this parameter for simplicity.

Reply:
Remove code only related to AzureStack,  while keep the ones relates to also custom cloud